source/manual/test-and-build-a-project-on-jenkins-ci.html.md

```diff
@@ -63,6 +63,31 @@ Many GOV.UK applications test against the
 To test your application for each PR on govuk-content-schemas, add it to the [govuk-content-schemas
 Jenkinsfile](https://github.com/alphagov/govuk-content-schemas/blob/master/Jenkinsfile).
 
+## Using Docker databases
+
+We have utilised Docker as a way to provide access to different versions of
+databases which can be used as part of your build. These can be accessed with
+the following connection strings:
+
+- MySQL 8: `mysql2://root:root@127.0.0.1:33068/<your-database-name>`
+- PostgreSQL 13: `postgresql://postgres@127.0.0.1:54313/<your-database-name>`
+
+Typically for these to work with a Rails application you'll need to set an
+environment variable. For example, in your Jenkinsfile, for an application
+that uses `TEST_DATABASE_URL`, you can set:
```

Comment:
It might also be nice to explicitly detail what the _default_ database versions are, and what will happen if you don't set the connection string.

```suggestion
- MySQL ?.? (default): `mysql2://root:root@127.0.0.1:3306/<your-database-name>`
- MySQL 8: `mysql2://root:root@127.0.0.1:33068/<your-database-name>`
- Postgres 9.6 (default): `postgresql://postgres@127.0.0.1:5432/<your-database-name>`
- Postgres 13: `postgresql://postgres@127.0.0.1:54313/<your-database-name>`

Typically for these to work with a Rails application you'll need to set an
environment variable. If no environment variable is present, the app
will use the default database version.

For example, in your Jenkinsfile, for an application
that uses `TEST_DATABASE_URL`, you can set:
```